theano/tensor/nnet/tests/test_abstract_conv.py

```diff
@@ -331,6 +332,47 @@ def tcase(self, i, f, s, b, flip, provide_shape, fd=(1, 1)):
                            check_trace=True, filter_dilation=fd)
 
 
+class TestAbstractConvNoOptim(BaseTestConv2d):
+    @classmethod
+    def setup_class(cls):
+        if theano.config.blas.ldflags == "":
+            raise SkipTest()
+        BaseTestConv2d.setup_class()
+        cls.inputs_shapes = [(8, 1, 6, 6)]
+        cls.filters_shapes = [(5, 1, 2, 2)]
+        cls.subsamples = [(1, 1), (2, 2)]
+        cls.filters_dilations = [(1, 1), (1, 2), (2, 1)]
+        cls.border_modes = ["valid", "full"]
+        cls.filter_flip = [True]
+        cls.provide_shape = [False]
+
+    def tcase(self, i, f, s, b, flip, provide_shape, fd=(1, 1)):
+        o = self.get_output_shape(i, f, s, b, fd)
+        if (not theano.config.blas.ldflags or
+                not theano.config.cxx or
+                theano.config.mode == "FAST_COMPILE"):
```

Comment:
Why the last condition? As you create your own mode, I don't think it is needed. Otherwise my 2 small comments, I think this PR is good. thanks